contracts/rewards/StandardRewards.sol

```diff
@@ -370,7 +370,8 @@ contract StandardRewards is IStandardRewards, ReentrancyGuardUpgradeable, Utils,
             isEnabled: true,
             startTime: startTime,
             endTime: endTime,
-            rewardRate: totalRewards / (endTime - startTime)
+            rewardRate: rewardRate,
+            remainingRewards: rewardRate * (endTime - startTime)
```

Comment:
Is your purpose here to get rid of the remainder in `totalRewards / (endTime - startTime)`?

If not, then you may as well keep the code as before and just add:
```
remainingRewards: totalRewards
```

Reply:
Yes, the rationale is to make sure that `remainingRewards` is equal to the rewards which are actually going to be distributed (`rewardRate * (endTime - startTime)`), which is usually a wei awei (🥁🥁) from `totalRewards`